src/client/testing/page-objects/change-site-details-pages/site-details.ts

```diff
@@ -131,23 +150,41 @@ export default class SiteDetailsPage extends RootPage {
     await expect(this.page.getByText(newInformation)).not.toBeVisible();
   }
 
-  async verifyDefaultSitePage() {
+  async verifyEditButtonNotVisible() {
+    await expect(this.editInformationCitizenButton).not.toBeVisible();
+    await expect(this.editSiteAttributesButton).not.toBeVisible();
+  }
+
+  async verifyEditButtonToBeVisible() {
+    await expect(this.editInformationCitizenButton).toBeVisible();
+    await expect(this.editSiteAttributesButton).toBeVisible();
+  }
+
+  async verifySitePage(
+    siteName: 'Church Lane Pharmacy' | 'Robin Lane Medical Centre',
+  ) {
+    const site = this.sites.find(x => x.name == siteName);
+
+    if (site === undefined) {
+      throw new Error();
+    }
```

Comment:
This is one of the reasons we shouldn't put seed data in page objects. If our seed data changed or we added extra sites, you'd have to change not only the lines above but also this method too. 
Ideally we only want to have to declare the seed data in one place then we only have to change it once in the future, if that makes sense?